src/pages/welcomePage.js

```diff
@@ -18,4 +19,5 @@ export const initWelcomePage = () => {
 
 const startQuiz = () => {
   initQuestionPage();
+  countdownNextPage();
```

Comment:
Maybe we can think of a better name for this.
How about `initCounter`?

Reply:
In case you like `initCounter`, we should rename all **countdown** keywords as counter across application I believe.